projects/plugins/backup/README.md

```diff
@@ -6,6 +6,19 @@ Easily restore or download a backup of your site from a specific moment in time.
 
 ### Installation From Git Repo
 
+The easiest way to get the latest version is by cloning the build plugin repo.
+
+1. `git clone https://github.com/Automattic/Jetpack-backup-plugin`
+2. Move this directory into the wp-content/plugins directory of your site.
+
+Alternatively, if you have made changes to the plugin or want to build from source:
+
+1. Check out and set up the Jetpack repo
+2. `cd projects/plugins/backup`
+3. `pnpm run build-production-concurrently` <-- this builds without fancy symlinks, so that the build can run standalone
+
+Now you can zip up the `projects/plugins/backup` directory and upload to your WordPress site.
+
```

Comment:
```suggestion
The easiest way to get the latest version by using the [Jetpack Beta plugin](https://jetpack.com/beta/).

Alternatively, the next is by grabbing the zip from the build plugin repo.

1. Download https://github.com/Automattic/jetpack-backup-plugin/archive/refs/heads/master.zip
2. Install via the wp-admin Add Plugin via upload method.

Lastly, if you have made changes to the plugin, you can either use the Jetpack Docker setup or the following to build from source:

1. Check out and set up the Jetpack repo, including the CLI.
2. `jetpack build plugins/backup --production` <-- this builds without fancy symlinks, so that the build can run standalone

Now you can zip up the `projects/plugins/backup` directory and upload to your WordPress site.

```